docs/design.md

```diff
@@ -12,7 +12,7 @@ The operator is primarily driven through an `UpgradeConfig` custom resource, whi
 
 The `UpgradeConfig` can be created directly on the cluster which is not managed via Hive. Also this method can be used for development/testing purposes
 
-For OpenShift Dedicated deployments, the `UpgradeConfig` is distributed via a [Hive SelectorSyncSet](https://github.com/openshift/hive/blob/master/docs/syncset.md) and managed by OpenShift SRE.
+For OpenShift Dedicated deployments, the `UpgradeConfig` is distributed via a [Hive SyncSet](https://github.com/openshift/hive/blob/master/docs/syncset.md) and managed by OpenShift SRE.
```

Comment:
We should get rid of this line entirely now - this hasn't been the case since late last year.

Reply:
Thanks for review @mrbarge.
I have updated the doc and removed the entire line from the doc.